scripts/dev/newReleaseBranch.py

```diff
@@ -556,24 +557,28 @@ def commit_all_and_push(project, message):
 
 def calculate_branch(branch_type, release_version=release_version, use_test_branches=args.test_branches):
     """Calculate the branch name
-    >>> calculate_branch("master", release_version="24.1.0")
-    'release/24.1'
+    >>> calculate_branch("master", release_version="23.2.0")
+    'release/23.2'
```

Comment:
Maybe this should just return `master`? It's confusing that it returns the same as `"hotfix"`, but different from `"service-pack"`.